src/data-source/esi/EsiError.ts

```diff
@@ -23,3 +8,26 @@ export enum EsiErrorKind {
   IO_ERROR = "esi:IOError",
   GENERIC_ERROR = "esi:Error",
 }
+
+/**
+ * Errors generated by fetchEndpoint().
+ */
+export class EsiError extends VError {
```

Comment:
Historical note: the dependency on VError was simply because swagger used VErrors and we wanted to be API-compatible during the switchover from swagger to fetchEndpoing(). The nested stack is useful, so I don't see a huge reason to depart from it right now, but just FYI.